tests/test_federation.py

```diff
@@ -46,10 +46,9 @@ def setUp(self):
         user_id = UserID("us", "test")
         our_user = create_requester(user_id)
         room_creator = self.homeserver.get_room_creation_handler()
+        config = {"preset": "public_chat"}
         self.room_id = self.get_success(
-            room_creator.create_room(
```

Comment:
This code was actually broken for perhaps a very long time, the public chat present was never being applied but since the auth check was monkeypatched this wasn't caught, but the changes I made exposed the bug so I fixed it.